packages/wdio-sauce-service/src/utils.js

```diff
@@ -0,0 +1,22 @@
+export function makeCapabilityFactory(tunnelIdentifier, options) {
+    return capability => {
+        const isW3C = !(capability.platform || capability.version) || capability['sauce:options']
```

Comment:
As mentioned before, let's use the `isW3C` helper from `@wdio/utils`

Reply:
Am looking at this at the moment, I'm having a weird issue though where the function isn't present if I use `import { isW3C } from '@wdio/utils` but is present if I use `import { isW3C } from '../../wdio-utils`... Is there someway that the build system could be caching the old version of `@wdio/utils`?

Reply:
> Is there someway that the build system could be caching the old version of @wdio/utils?

You have to export the method in the `@wdio/utils` package.

Reply:
> You have to export the method in the @wdio/utils package.

I think I have? See https://github.com/webdriverio/webdriverio/pull/5610/commits/c1430a61d8aa6395a19d1a8b5bff00276627ed94

Reply:
Tests are still failing with `utils.isW3C is not a function` though :(